README.md

```diff
@@ -56,6 +56,11 @@ For a more detailed overview and getting started guide, please see
      * [\__typename](#__typename)
      * [\_x_count](#_x_count)
   * [The GraphQL schema](#the-graphql-schema)
+  * [Query cost estimation](#query-cost-estimation)
```

Comment:
I would move this to under the `SQL` section. It's hard to talk about cost estimation when we haven't even described what the execution model is.

Also, consider using `Title Case`. I know we don't consistently use it, but we should -- all of these are section and subsection titles.